src/components/DateInput/styles.tsx

```diff
@@ -0,0 +1,50 @@
+import styled from 'styled-components';
+import { Open_Sans as OpenSans } from 'next/font/google';
+import COLORS from '@/styles/colors';
+import { H4 } from '@/styles/text';
+
+const openSans = OpenSans({ subsets: ['latin'] });
+
+const getBorderColor = (error: boolean, filled: boolean): string => {
+  if (error) {
+    return COLORS.redMid;
+  }
+  if (filled) {
+    return COLORS.greyMid;
+  }
+  return COLORS.greyLight;
+};
+
+export const InputLabel = styled(H4)`
+  color: ${COLORS.greyDark};
+  margin-bottom: 0.625rem; //10px
+`;
+
+export const InputDate = styled.input<{ $error: boolean; $filled: boolean }>`
+  ${openSans.style}
+  font-size: 1rem;
+  font-weight: 400;
+  color: ${props =>
+    props.$filled ? COLORS.greyDarker : COLORS.placeholderText};
+  border-radius: 0.313rem; // 5px
+  border: 2px solid ${({ $error, $filled }) => getBorderColor($error, $filled)};
```

Comment:
nit: can we stay consistent using either `props` or destructuring? (`props => ...` vs `({ $val }) => ...`)

Reply:
Note [worksesh]: moving forward we should go with the destructuring approach